modules/cluster-logging-collector-pod-location.adoc

```diff
@@ -6,43 +6,24 @@
 [id="cluster-logging-collector-pod-location_{context}"]
 = Viewing logging collector pods
 
-You can view the Fluentd logging collector pods and the corresponding nodes that they are running on. The Fluentd logging collector pods run only in the `openshift-logging` project.
+You can view the logging collector pods and the corresponding nodes that they are running on. The logging collector pods run only in the `openshift-logging` project.
```

Comment:
that is not correct now.

Reply:
Updated to remove the name of the project and the outdated sentence.